### PR TITLE
Renamed icatypes.PortPrefix to icatypes.ControllerPortPrefix 

### PR DESCRIPTION
**Pull request by alizahidraja**
## Description

Renamed icatypes.PortPrefix to icatypes.ControllerPortPrefix & icatypes.PortID to icatypes.HostPortID
closes: #2433

---

Before we can merge this PR, please make sure that all the following items have been
checked off. If any of the checklist items are not applicable, please leave them but
write a little note why.

- [x] Targeted PR against correct branch (see [CONTRIBUTING.md](https://github.com/cosmos/ibc-go/blob/master/CONTRIBUTING.md#pr-targeting))
- [x] Linked to Github issue with discussion and accepted design OR link to spec that describes this work.
- [x] Code follows the [module structure standards](https://github.com/cosmos/cosmos-sdk/blob/master/docs/building-modules/structure.md).
- [ ] Wrote unit and integration [tests](https://github.com/cosmos/ibc-go/blob/master/CONTRIBUTING.md#testing) (already written)
- [ ] Updated relevant documentation (`docs/`) or specification (`x/<module>/spec/`) (not applicable)
- [x] Added relevant `godoc` [comments](https://blog.golang.org/godoc-documenting-go-code).
- [x] Added a relevant changelog entry to the `Unreleased` section in `CHANGELOG.md`
- [x] Re-reviewed `Files changed` in the Github PR explorer
- [x] Review `Codecov Report` in the comment section below once CI passes

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -93,6 +93,7 @@ Ref: https://keepachangelog.com/en/1.0.0/
 * (light-clients/06-solomachine) Moving `verifyMisbehaviour` function from update.go to misbehaviour_handle.go.
 * (apps/27-interchain-accounts) [\#2297](https://github.com/cosmos/ibc-go/pull/2297) Adding cli command to generate ICS27 packet data.
 * (modules/core/keeper) [\#1728](https://github.com/cosmos/ibc-go/pull/2399) Updated channel callback errors to include portID & channelID for better identification of errors.
+* (modules/apps/27-interchain-accounts) [\#2433](https://github.com/cosmos/ibc-go/pull/2450) Renamed icatypes.PortPrefix to icatypes.ControllerPortPrefix & icatypes.PortID to icatypes.HostPortID
```
Comment (crodriguezvega) [inline CHANGELOG.md:96]: I believe this change is API breaking? So maybe we should move the log to that section?

**Review by crodriguezvega — COMMENTED** [review]
Thank you, @alizahidraja!

File: CHANGELOG.md (hunk 1 above)
Comment (alizahidraja) [reply CHANGELOG.md:96]: Oh gotcha, will do

**Review by crodriguezvega — APPROVED** [review]

**Review by colin-axner — APPROVED** [review]
Ay! Thanks @alizahidraja

**Pull request merged**